test/e2e.js

```diff
@@ -0,0 +1,29 @@
+var spawn = require("child_process").spawn;
```

Comment:
Can you change this declaration of `var` to `const` to make the code base consistent, since we have used either `const` or `let` in this project for variable declarations.